src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -95,7 +95,14 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
     String recipient = request.getParameter("recipient");
-    
+
+    text = text.replace("[b]", "<strong>").replace("[/b]","</strong>");
```

Comment:
Oh, also (and I'm just being picky now) there should be a space after each comma in these lines.

Reply:
Hi Dan, I made the changes committed them to this branch